packages/cli/src/utils/browserManagement/index.ts

```diff
@@ -265,6 +305,7 @@ export class BrowserManagement {
     await Promise.all(
       urls.map(async (url) => {
         await this.navigateAndScroll(url);
+        await this.clickOnAcceptBanner(url);
```

Comment:
Lets refactor this routine a little bit.
- Make different function for navigating to the page and scrolling it.
- After that we can call the routines in the following manner.
  - Navigate ( This navigates to the page and waits for network stream to get idle )
  - Click on Accept banner.
  - Wait for `this.pageWaitTime / 2` time.
  - Scroll the page to the end.
  - Wait for `this.pageWaitTime / 2` time.

Reply:
Resolved in latest commits.